### ISSUE TITLE
License terms 

### ISSUE THREAD

**Issue by GenevieveBuckley** [mandeepbhutani/cookiecutter-pyqt5#3]
I think the README needs some discussion of the pyqt license terms, since if the user doesn't have a commercial license (and is using pyqt under GPL v3 instead) they will need to update the license for their own project to match.

> PyQt5 is released under the GPL v3 license and under a commercial license that allows for the development of proprietary applications.
https://pypi.org/project/PyQt5/

https://www.riverbankcomputing.com/commercial/license-faq

**Comment by mandeep** [comment]
Hi @GenevieveBuckley! Thanks for bringing this up. I agree that the PyQt5 license can be tricky so I'll update the project README to include license details.

**Comment by GenevieveBuckley** [comment]
Yeah, I've just been trying to wrap my head around it too.  I'll chime in with a couple of thoughts on your open PR.

**Comment by mandeep** [comment]
@GenevieveBuckley Oops! I went ahead and merged it once the checks passed. Feel free to add more comments here if you would like me to add more details in the README. You are more than welcome to submit a PR if you wish.

**Comment by GenevieveBuckley** [comment]
I must have been midway through typing. Comment:
https://github.com/mandeep/cookiecutter-pyqt5/pull/4/files#r272882021

tl;dr
There needs to be a sentence that says "If you do not have a commercial license for PyQt, you must update the license for your project to [GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html) (a copyleft license) as required by the  licensing terms of PyQt5."
